include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -855,6 +925,38 @@ class FEEvaluationBase
    */
   unsigned int cell;
 
+  /**
+   * Flag holding information whether a face is on the left or the right of a
```

Comment:
> on the left or the right

was not sure what you mean. Also the name suggests that it's boundary / non-boundary flag.